python/lsst/ts/ofc/controllers/base_controller.py

```diff
@@ -173,44 +199,64 @@ def aggregated_state(self):
         """
         return self.dof_state[self.ofc_data.dof_idx]
 
+    def reset_history(self) -> None:
+        """Reset the history of the controller."""
+        self.dof_state0 = self.dof_state
+        self.integral = (
+            self.setpoint[self.ofc_data.dof_idx] - self.dof_state[self.ofc_data.dof_idx]
+        )
+        self.previous_error = (
+            self.setpoint[self.ofc_data.dof_idx] - self.dof_state[self.ofc_data.dof_idx]
+        )
```

Comment:
I am not sure this part of logic. Do you have any (unit) test to verify this? Or for example, run a simulation first. Then, call this function to reset the history, followed by a new simulation to make sure everything in PID will start over. This would be important because we would turn the PID parameters in `ts_mtaos`. We need to make sure before the new turning, we have cleared the history.